index.js

```diff
@@ -18,10 +20,13 @@ module.exports = function (opts) {
 			return;
 		}
 
+		var target = path.relative(opts.abspath, file.path);
```

Comment:
hmm.  the problem is that ``file.path`` is an absolute path itself and vulcanize expects a target relative to ``abspath``.